.travis.yml

```diff
@@ -167,24 +167,23 @@ script:
     - install_run $PLAT
 
 after_success:
-    # trigger an upload to the shared ecosystem
-    # infrastructure at: https://anaconda.org/scipy-wheels-nightly
-    # for cron jobs only (restricted to master branch once
-    # per week)
-    # SCIPY_WHEELS_NIGHTLY is a secret token
-    # used in Travis CI config, originally
-    # generated at anaconda.org for scipy-wheels-nightly
-    - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+    # upload wheels for pushes and merges.
+    - if [ "$TRAVIS_BRANCH" == "master" ]; then
+          # Pushes and merges to master we use the scipy-wheels-nightly.
+          # SCIPY_WHEELS_NIGHTLY is a secret token
+          # used in Travis CI config, originally
+          # generated at anaconda.org for scipy-wheels-nightly
           ANACONDA_ORG="scipy-wheels-nightly";
-          pip install git+https://github.com/Anaconda-Platform/anaconda-client.git@ce89e4351eef;
-          anaconda -t ${SCIPY_WHEELS_NIGHTLY} upload --force -u ${ANACONDA_ORG} ${TRAVIS_BUILD_DIR}/wheelhouse/*.whl;
-      fi
-    # for merges (push events) we use the staging area instead;
-    # SCIPY_STAGING_UPLOAD_TOKEN is a secret token used in Travis
-    # CI config, originally generated at anaconda.org for
-    # multibuild-wheels-staging
-    - if [ "$TRAVIS_EVENT_TYPE" == "push" ]; then
+          TOKEN=${SCIPY_WHEELS_NIGHTLY};
+      else
+          # for merges (push events) we use the staging area instead;
```

Comment:
With `TRAVIS_EVENT_TYPE` filter removed, that means we do not block the `pull_request` event for backport activity in wheels repo.

I'd have to double check to be sure, but the secrets may be restricted to branches originating in the repo proper instead of from forks, which may possibly cause Travis CI to error (and perhaps even explain the lack of a CI matrix entry for Travis here?).

More generally though, I'm at least equally confused about the new settings vs. the old ones. I just checked, and I don't actually see Travis CI cron jobs running for SciPy wheels at the moment/last 6 months, perhaps because Ralf and I might have decided to purge the cron when the job credits stuff happened? (there is no cron job in our wheels repo settings for Travis at the moment).